todo0/src/server/controllers/oidc.ts

```diff
@@ -128,22 +127,32 @@ function createStrategy(username: string) {
   );
 }
 
-controller.post('/signout', async (req, res, next) => {
-  req.logout((err) => {
-    if (err) {
-      next(err);
-    }
-  });
-
-  req.session.destroy((err) => {
-    if (!err) {
-      res.status(200).clearCookie(TODO_COOKIE_NAME, { path: '/' }).json({ status: 'Success' });
-    } else {
-      next(err);
-    }
-  });
-});
-
+controller.post(
+  '/signout',
+  async (req, res, next) => {
+    req.session.destroy((err) => {
+      if (!err) {
+        res.clearCookie(TODO_COOKIE_NAME, { path: '/' });
+        res.clearCookie('connect.sid', { path: '/' });
```

Comment:
do we need to clear these additional cookies?  Aren't they owned by the auth server?

Reply:
yah it probably wont work in production since the domains are different